nonnull_test.go

```diff
@@ -1041,7 +1045,7 @@ func TestNonNull_NullsTheTopLevelIfSyncNonNullableFieldErrors(t *testing.T) {
       query Q { nonNullPromise }
 	`
 	expected := &graphql.Result{
-		Data: map[string]interface{}{"nonNullPromise": nil},
+		Data: nil,
```

Comment:
Interesting that this changes the resulting data... hopefully that won't be an issue, since this only occurs on an error anyway.